macosx/TorrentCell.mm

```diff
@@ -296,34 +309,35 @@ - (void)drawInteriorWithFrame:(NSRect)cellFrame inView:(NSView*)controlView
     [self drawBar:minimal ? [self barRectMinForBounds:cellFrame] : [self barRectRegForBounds:cellFrame]];
 
     //group coloring
-    NSRect const iconRect = [self iconRectForBounds:cellFrame];
+    NSRect const parentRect = [self iconRectForBounds:cellFrame];
+    NSRect iconRect = NSMakeRect(parentRect.origin.x, parentRect.origin.y, parentRect.size.width, parentRect.size.height);
+    iconRect.origin.x += minimal ? 0 : GROUP_INSET_REG * 3;
 
     NSInteger const groupValue = torrent.groupValue;
     if (groupValue != -1)
     {
-        NSRect groupRect = NSInsetRect(iconRect, -1.0, -2.0);
+        NSRect groupRect = NSInsetRect(parentRect, -1.0, -2.0);
         if (!minimal)
         {
-            groupRect.size.height -= 1.0;
-            groupRect.origin.y -= 1.0;
+            groupRect.size.height = cellFrame.size.height - (GROUP_INSET_REG * 4);
+            groupRect.size.width = GROUP_INSET_REG;
+            groupRect.origin.y -= GROUP_INSET_REG * 2;
+            groupRect.origin.x += GROUP_INSET_REG;
         }
-        CGFloat const radius = minimal ? 3.0 : 6.0;
+        CGFloat const radius = 2.0;
 
+        CGFloat fractionOfBlendedColor = 0.05;
```

Comment:
New blend factor is 4 times less. Why?

Reply:
no specific reason,
I think it looks better